crates/iota-indexer/src/config.rs

```diff
@@ -256,6 +262,8 @@ pub mod deprecated {
         pub db_name: Option<String>,
         #[arg(long, default_value = "http://0.0.0.0:9000", global = true)]
         pub rpc_client_url: String,
+        #[arg(long, default_value = None, global = true)]
+        pub grpc_client_url: Option<String>,
```

Comment:
This config here is deprecated, don't add it here, only to `IngestionSources`.

Reply:
Thanks for identifying this. Removed it at [91b6d47](https://github.com/iotaledger/iota/pull/7511/commits/91b6d47722d3add8034d12a5834c2a432ae94aa2).